wagtail/admin/views/pages.py

```diff
@@ -345,6 +346,14 @@ def edit(request, page_id):
     user_has_lock = page.locked_by_id == request.user.id
     page_locked = page_perms.page_locked()
 
+    if user_has_lock:
+        messages.warning(request, format_html(_("<b>Page '{}' was locked</b> by <b>you</b> on <b>{}</b>."), page.get_admin_display_title(), page.locked_at.strftime("%d %b %Y %H:%M")), extra_tags='lock')
+    if page_locked:
+        locked_by_message = ""
+        if page.locked_by:
+            locked_by_message = format_html(" by <b>{}</b>", str(page.locked_by))
```

Comment:
We probably want to write out the whole message again here, since sentence structure could vary between languages

Reply:
Good point - have added.